doc/src/cylc-user-guide/cug.tex

```diff
@@ -5208,6 +5208,51 @@ \subsubsection{Custom Jinja2 Filters}
     \item \lstinline={{'1066/10/14 08:00:00' | strftime('%Y%m%dT%H', '%Y/%m/%d %H:%M:%S')}}= - 10661014T08
 \end{myitemize}
 
+\paragraph{duration\_to\_hours}
+
+The ``duration\_to\_hours'' filter can be used to format ISO8601 duration
+strings as a floating-point number of hours.
+
+\lstset{language=suiterc}
+\begin{lstlisting}
+{% set CYCLE_INTERVAL = 'PT1D' %}
+{{ CYCLE_INTERVAL | duration_to_hours }}  # 24.0
+{% set CYCLE_SUBINTERVAL = 'PT30M' %}
+{{ CYCLE_SUBINTERVAL | duration_to_hours }}  # 0.5
+\end{lstlisting}
+
+While the filtered value is a floating-point number, it is often required to
+supply an integer to suite entities (e.g. environment variables) that require
```

Comment:
Need `e.g.\ ` (i.e. escape the space after e.g. to avoid interpretation as end-of-sentence, which makes the space bigger.)